ui/shared/status/StatusChatInput.qml

```diff
@@ -209,6 +226,26 @@ Rectangle {
                 emojiSuggestions.close()
             }
         }
+
+        let shiftOrCtrlModifier = (event.modifiers & Qt.ShiftModifier) || (event.modifiers & Qt.ControlModifier) || (event.modifiers & Qt.MetaModifier)
+        let arrowKeys = event.key == Qt.Key_Left || event.key == Qt.Key_Right || event.key == Qt.Key_Up || event.key == Qt.Key_Down
+
+        if (event.key !== Qt.Key_Backspace &&
+            event.key !== Qt.Key_Enter &&
+            event.key !== Qt.Key_Return &&
+            event.key !== Qt.Key_Tab &&
+            event.key !== Qt.Key_Shift &&
+            event.key !== Qt.Key_Ctrl &&
+            event.key !== Qt.Key_Copy &&
+            event.key !== Qt.Key_Cut &&
+            event.key !== Qt.Key_Select &&
+            event.key !== Qt.Key_Undo &&
+            event.key !== 16777249 &&
+            !(shiftOrCtrlModifier && arrowKeys) &&
+            !shiftOrCtrlModifier &&
+            !arrowKeys) {
```

Comment:
This is a lot of conditions. Would it be possible to have a comment above to say what it is checking for, because even now, I'm not sure exactly what it is doing. Shouldn't we just be parsing when the last character was a star, back-tick or tilde?